config.sh

```diff
@@ -16,6 +16,7 @@ export coupled='ATM_OCN_ICE' # NO or ATM_OCN_ICE
 export cores=`expr $NODES \* $corespernode`
 
 export do_cleanup='false' # if true, create tar files, delete *mem* files.
+export days_keep=1
```

Comment:
add a comment here like 'clean up all files earlier than now - days_keep'

Reply:
Added: 
export days_keep=1  # if save_hpss="true", delete local directory copy for the cycle (current_date - days_keep)
                    # set to 0 to turn off deleting of past directories